src/browser/BrowserService.cpp

```diff
@@ -487,19 +490,23 @@ void BrowserService::addEntry(const QString& id,
         config.setRealm(realm);
     }
     config.save(entry);
+    return ReturnValue::Success;
 }
 
-void BrowserService::updateEntry(const QString& id,
-                                 const QString& uuid,
-                                 const QString& login,
-                                 const QString& password,
-                                 const QString& url,
-                                 const QString& submitUrl)
+BrowserService::ReturnValue
+BrowserService::updateEntry(const QString& id,
+                            const QString& uuid,
+                            const QString& login,
+                            const QString& password,
+                            const QString& url,
+                            const QString& submitUrl)
 {
+    ReturnValue result = ReturnValue::Error;
```

Comment:
Same comment

Reply:
This is actually returned. See line 575.